CHANGELOG.md

```diff
@@ -1,217 +1,241 @@
-# Release 8.0.0
+# Release 9.0.0
 
 **NOTE:**
-_This changelog only contains release notes for PWA Studio 8.0.0._
+_This changelog only contains release notes for PWA Studio 9.0.0._
 _For older release notes, see [PWA Studio releases][]._
 
 ## Table of contents
 
--   [What's new in 8.0.0](#whats-new-in-800)
+-   [What's new in 9.0.0](#whats-new-in-900)
 -   [Pull requests merged in this release](#pull-requests-merged-in-this-release)
 -   [Known issues](#known-issues)
 -   [Upgrading from a previous version](#upgrading-from-a-previous-version)
 
-## What's new in 8.0.0
+## What's new in 9.0.0
 
-PWA Studio 8.0.0 contains new features, refactors, and various improvements.
+PWA Studio 9.0.0 contains new features, refactors, and various improvements.
 
-### Improved performance
+### Extensibility framework improvements
 
-A lot of work has been done in this release to improve storefront performance.
-This means that any project using the latest PWA Studio components will benefit from these updates.
+This releases adds a few improvements to the extensibility framework in PWA Studio to make it easier for developers to customize their storefronts.
+For an overview of this framework, check out the new [Extensibility framework][] topic on the docs site.
 
-One of the bigger changes is the migration to [Apollo Client 3.0][].
-This version of the Apollo Client provides better cache controls and better network fetching performance in general.
+In previous releases, Peregrine talons had limited Target coverage.
+This release adds an automatic API generator to Peregrine that exposes all hooks and talons as Targets.
+This means that existing and future hooks and talons in Peregrine automatically get their own Targets that developers may use for modifying or extending functionality.
 
-This release also includes various refactors and improvements on the GraphQL queries themselves to reduce the API request times from Magento.
+This release also adds the _Targetables_ feature to the extensibility framework.
+These represent source files used in your PWA Studio project, and
+they give developers the ability to change the source code during the build process.
+This means that developers no longer have to copy PWA Studio source code into their storefront projects to make minor modifications.
 
-[apollo client 3.0]: https://www.apollographql.com/docs/react/migrating/apollo-client-3-migration/
+[extensibility framework]: <https://pwastudio.io/pwa-buildpack/extensibility-framework/>
 
-### Complete cart and checkout experience
+### PWA Studio extensions
 
-This release finishes the full page cart and checkout features introduced in previous releases.
-The complete cart and checkout workflow is based on research made by members of the UX team.
+PWA Studio's extensibility framework lets developers create extensions and install them as project dependencies in their storefronts.
+This release converts some existing Venia features into PWA Studio extensions.
```

Comment:
I think this could sound better

> As part of the work on the new extensibility framework we have moved a number of existing Venia features into PWA Studio extensions.